wrangler-core/src/main/java/io/cdap/directives/parser/ParseAvroFile.java

```diff
@@ -105,7 +115,10 @@ public void destroy() {
           }
         } else {
           throw new DirectiveExecutionException(
-            NAME, String.format("Column '%s' is of invalid type. It should be of type 'byte array'.", column));
+            NAME, String.format("Avro data file parsing directive requires '%s' to be of byte. " +
+                                  "Change type in input as byte and make sure the format in File source is " +
+                                  "set as 'blob'. We highly recommend to use File source connector to read Avro files.",
```

Comment:
We recommend using the avro format in the file source instead of using this directive.